tuberculosis/test/widget_test.dart

```diff
@@ -5,14 +5,20 @@
 // are correct.
 
 import 'package:Tubuddy/main.dart';
+import 'package:Tubuddy/pages/login_page.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
```

Comment:
moet dit erin of niet?

Reply:
(de commented code)

Reply:
Die test is kapot door translations dus nee maar als iemand tijd heeft dan mag die hem fixen.